sdk/go/dagger.gen.go

```diff
@@ -2859,7 +2859,7 @@ func (r *Directory) WithFiles(path string, sources []*File, opts ...DirectoryWit
 // DirectoryWithNewDirectoryOpts contains options for Directory.WithNewDirectory
 type DirectoryWithNewDirectoryOpts struct {
 	// Permission granted to the created directory (e.g., 0777).
-	Permissions int
+	Permissions int `default:"420"`
```

Comment:
:sob: just realized the weirdness of this... not too much of an issue, since we have the same problem in our graphql schema, and in our other sdks.

But would be cool to maybe attach *more* directives that could act as hints to display this as octal/hex/etc.

Reply:
Hah yeah unfortunate. These octal numbers are annoying to deal with but felt like the most intuitive DX for these APIs. I remember we brielfy considered something like "rw-rw----" but that's just not what people usually interface with in APIs like this, and anything too declarative would probably stray too far from common convention/memory.